src/analyses/goto_check.cpp

```diff
@@ -1860,6 +1935,8 @@ void goto_checkt::goto_check(
         flag_resetter.set_flag(enable_undefined_shift_check, false);
       else if(d.first == "disable:nan-check")
         flag_resetter.set_flag(enable_nan_check, false);
+      else if(d.first == "disable:pointers-in-assumptions-check")
```

Comment:
Why a different name here?
```suggestion
      else if(d.first == "disable:pointer-primitive-check")
```